plugin/core/windows.py

```diff
@@ -423,13 +423,13 @@ def on_post_exit_async(self, session: Session, exit_code: int, exception: Option
             listener.on_session_shutdown_async(session)
         if exit_code != 0 or exception:
             config = session.config
-            msg = "{} exited with status code {}".format(config.name, exit_code)
-            if exception:
-                msg += " and message:\n\n---\n{}\n---".format(str(exception))
-            msg += "".join((
-                "\n\nDo you want to restart {0}?\n\nIf you choose Cancel, {0} will be disabled for this project. ",
+            msg = "".join((
+                "{0} exited with status code {1}. ",
+                "Do you want to restart {0}? If you choose Cancel, {0} will be disabled for this window. ",
                 "Enable {0} again by running \"LSP: Enable Language Server In Project\" from the Command Palette."
```

Comment:
Again server name repeated 4 times.

```suggestion
                "{0} exited with status code {1}. ",
                "Do you want to restart it? If you choose Cancel, it will be disabled for this window. ",
                "Re-enable by running \"LSP: Enable Language Server In Project\" from the Command Palette."
```